packages/exa-git/exa-git.pacscript

```diff
@@ -15,7 +20,6 @@ build() {
 }
 
 install() {
-    sudo mkdir -p /usr/src/pacstall/exa-git/usr/bin
-    sudo mv target/release/exa /usr/src/pacstall/exa-git/usr/bin
-    sudo chmod +x /usr/src/pacstall/exa-git/usr/bin/exa
+    sudo install -Dm755 target/release/exa -t "${pkgdir}"/usr/bin
+    sudo chmod +x "${pkgdir}"/usr/bin/exa
```

Comment:
Why are you `chmod`ding it after `-m755`?

Reply:
Oops, forgot to remove it